### ISSUE TITLE
Kubernetes is unable to execute from Ansible AWX

### ISSUE THREAD

**Issue by ghost** [ansible/awx#3208]
@AlanCoding @ryanpetrello @willthames @chouseknecht 

##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME

 - UI

##### SUMMARY

Ansible AWX Kubernetes Jobs/yaml is unable to execute.

##### ENVIRONMENT
* AWX version: 3.0.0
* AWX install method: docker on linux
* Ansible version:  Ansible 2.7.4
* Operating System:
  NAME="Ubuntu"
  VERSION="18.04.1 LTS (Bionic Beaver)"
* Web Browser: Safari Web Browser from MacOS

##### STEPS TO REPRODUCE

```
kylix3511 >>>> cat fetch_all_services.yaml 

---
- hosts: localhost
  connection: local
  vars:
      ansible_python_interpreter: '{{ ansible_playbook_python }}'

  tasks:
    - python_requirements_facts:
         dependencies:
            - openshift
            - requests

    - name: Get a list of all service objects
      k8s_facts:
        api_version: v1
        kind: Service
        namespace: "kylix3511"
      register: service_list
kylix3511 >>>>

```
##### EXPECTED RESULTS

Show return all the services in the namespace "kylix3511"

##### ACTUAL RESULTS

[job_125.txt](https://github.com/ansible/awx/files/2852710/job_125.txt)

##### ADDITIONAL INFORMATION

I follow the https://github.com/ansible/awx/blob/devel/docs/custom_virtualenvs.md#preparing-a-new-custom-virtualenv to install openshift in the ***docker awx-task container*** still I have same issue and same error ...  ***seems more direction or documentation needed for AWX on Docker virtual env. ***

```
root@kylix3511.mylabserver.com:/root$docker exec -it 43d23a720cb8 bash 
[root@awx awx]# ps -ef | grep -i task 
root         1     0  0 Feb04 ?        00:00:06 /tini -- /bin/sh -c /usr/bin/launch_awx_task.sh
root         8     1  0 Feb04 ?        00:00:00 bash /usr/bin/launch_awx_task.sh
root       147     8  0 Feb04 ?        00:04:20 /usr/bin/python2 /usr/bin/supervisord -c /supervisor_task.conf
root      7702  7684  0 19:20 pts/1    00:00:00 grep --color=auto -i task
[root@awx awx]# pip search kubernetes  | grep -i installed
 DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
  INSTALLED: 8.0.1 (latest)
[root@awx awx]# pip search openshift  | grep -i installed
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
  INSTALLED: 0.8.4 (latest)
[root@awx awx]# 
[root@awx awx]# 
[root@awx awx]# pip show openshift
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
Name: openshift
Version: 0.8.4
Summary: OpenShift python client
Home-page: https://github.com/openshift/openshift-restclient-python
Author: OpenShift
Author-email: UNKNOWN
License: Apache License Version 2.0
Location: /usr/lib/python2.7/site-packages
Requires: dictdiffer, jinja2, kubernetes, python-string-utils, ruamel.yaml, six
Required-by: 
[root@awx awx]# pip show kubernetes
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
Name: kubernetes
Version: 8.0.1
Summary: Kubernetes python client
Home-page: https://github.com/kubernetes-client/python
Author: Kubernetes
Author-email: UNKNOWN
License: Apache License Version 2.0
Location: /usr/lib/python2.7/site-packages
Requires: ipaddress, websocket-client, setuptools, certifi, pyyaml, requests-oauthlib, six, adal, requests, python-dateutil, google-auth, urllib3
Required-by: openshift
[root@awx awx]# 

```

**Comment by matburt** [comment]
> IOError: [Errno 2] No such file or directory: '/var/lib/awx/.kube/config'

Looks like you aren't authenticated

**Comment by ryanpetrello** [comment]
If you're seeing the above `IOError`, I suspect that `/var/lib/awx/.kube` is being blocked by bubblewrap.  See https://docs.ansible.com/ansible-tower/latest/html/administration/proot_func_variables.html#bubblewrap-functionality-and-variables for details.

**Comment by ghost** [comment]
Hi @ryanpetrello 

I have already done the ... Paths to Expose to Isolated Jobs  as AWX_PROOT_SHOW_PATHS  = /var/lib/awx/.kube

![image](https://user-images.githubusercontent.com/38637223/52593241-548ccc00-2dfd-11e9-8781-0302b5d08324.png)

Still the same error ... 

By the way, I don't have option or choice or way to Enable the Job Isolation from OFF to ON. And I’m getting the error message …. “This setting has been set manually in a setting file and is now disabled. 


![image](https://user-images.githubusercontent.com/38637223/52593710-82264500-2dfe-11e9-8511-baf772b4d712.png)

**Comment by ryanpetrello** [comment]
If you've got bwrap turned off entirely (which seems to be the case from your screenshot), and you're still seeing `IOError: [Errno 2] No such file or directory: '/var/lib/awx/.kube/config'`, then I agree with @matburt, it sounds like your task container doesn't have kube credentials configured properly.
